.github/workflows/bindings.yml

```diff
@@ -320,16 +333,23 @@ jobs:
       - name: build & install iOS Interop client
         run: |
           cd interop/src/clients/InteropClient
+
+          echo "INTEROP_SIMULATOR_DEVICE_1=$(xcrun simctl create "iPhone 16 e2e-interop-test" \
+                    com.apple.CoreSimulator.SimDeviceType.iPhone-16 \
+                    com.apple.CoreSimulator.SimRuntime.iOS-18-6)" >> $GITHUB_ENV
           xcodebuild -scheme InteropClient -sdk iphonesimulator -configuration Release \
                      -destination 'platform=iOS Simulator,name=iPhone 16' clean build install DSTROOT=./Products
-          echo "INTEROP_SIMULATOR_DEVICE=$(./create_simulator.sh)" >> $GITHUB_ENV
+          echo "INTEROP_SIMULATOR_DEVICE_2=$(./create_simulator.sh)" >> $GITHUB_ENV
```

Comment:
Why do we need two simulator devices?

Reply:
The interop test communicates between two devices I think? This is nothing that changed with this PR